### PR TITLE
Clarify the FOV setting in the 3D editor camera is vertical FOV

### PR DESCRIPTION
**Pull request by Calinou**
*Happy new year* :slightly_smiling_face:

## Preview

There's a tooltip if you hover the text field:

image

### CODE REVIEW

**Comment by ryevdokimov** [comment]
It should be clarified here as well: 
image

**Comment by Calinou** [comment]
> It should be clarified here as well:

Done :slightly_smiling_face:

File: doc/classes/EditorSettings.xml [hunk 1]
```diff
@@ -262,7 +262,8 @@
...
 		</member>
 		<member name="editors/3d/default_fov" type="float" setter="" getter="">
-			The default camera field of view to use in the 3D editor (in degrees). The camera field of view can be adjusted on a per-scene basis using the [b]View[/b] menu at the top of the 3D editor. If a scene had its camera field of view adjusted using the [b]View[/b] menu, this setting is ignored in the scene in question. This setting is also ignored while a Camera3D node is being previewed in the editor.
+			The default camera vertical field of view to use in the 3D editor (in degrees). The camera field of view can be adjusted on a per-scene basis using the [b]View[/b] menu at the top of the 3D editor. If a scene had its camera field of view adjusted using the [b]View[/b] menu, this setting is ignored in the scene in question. This setting is also ignored while a Camera3D node is being previewed in the editor.
+			[b]Note:[/b] The editor camera always uses the Keep Height aspect mode.
```
Comment (Mickeon) [inline doc/classes/EditorSettings.xml:266]: It's not exactly clear what this is referring to at a glance. May be worth surrounding this in quotes? 
```suggestion
			[b]Note:[/b] The editor camera always uses the "Keep Height" aspect mode.
```

File: doc/classes/EditorSettings.xml [hunk 2]
```diff
@@ -262,7 +262,8 @@
...
 		</member>
 		<member name="editors/3d/default_fov" type="float" setter="" getter="">
-			The default camera field of view to use in the 3D editor (in degrees). The camera field of view can be adjusted on a per-scene basis using the [b]View[/b] menu at the top of the 3D editor. If a scene had its camera field of view adjusted using the [b]View[/b] menu, this setting is ignored in the scene in question. This setting is also ignored while a Camera3D node is being previewed in the editor.
+			The default camera vertical field of view to use in the 3D editor (in degrees). The camera field of view can be adjusted on a per-scene basis using the [b]View[/b] menu at the top of the 3D editor. If a scene had its camera field of view adjusted using the [b]View[/b] menu, this setting is ignored in the scene in question. This setting is also ignored while a Camera3D node is being previewed in the editor.
```
Comment (Mickeon) [inline doc/classes/EditorSettings.xml:265]: While you're here:
```suggestion
			The default camera vertical field of view to use in the 3D editor (in degrees). The camera field of view can be adjusted on a per-scene basis using the [b]View[/b] menu at the top of the 3D editor. If a scene had its camera field of view adjusted using the [b]View[/b] menu, this setting is ignored in the scene in question. This setting is also ignored while a [Camera3D] node is being previewed in the editor.
```

**Review by Mickeon — APPROVED** [review]
It's good to know!

File: editor/plugins/node_3d_editor_plugin.cpp [hunk 3]
```diff
@@ -8576,7 +8576,8 @@ Node3DEditor::Node3DEditor() {
 	settings_fov->set_step(0.1);
 	settings_fov->set_value(EDITOR_GET("editors/3d/default_fov"));
 	settings_fov->set_select_all_on_focus(true);
-	settings_vbc->add_margin_child(TTR("Perspective FOV (deg.):"), settings_fov);
+	settings_fov->set_tooltip_text(TTR("The editor camera always uses the Keep Height aspect mode, so FOV is defined as a vertical value."));
```
Comment (Mickeon) [inline editor/plugins/node_3d_editor_plugin.cpp:8579]: Forgot to insert in the review. The latter sentence is much more "of interest" to the user than the former.
```suggestion
	settings_fov->set_tooltip_text(TTR("FOV is defined as a vertical value. The editor camera always uses the Keep Height aspect mode."));
```
Or similar

**Comment by ryevdokimov** [comment]
I'm not my reviews show up for anyone, so I'll leave a comment. Sorry if it's redundant. 

I think the lower case "v" in vertical is a bit odd, and inconsistent with the rest of the editor. Making it capital or saying VFOV (which is an appropriate term) would look better IMO.

**Comment by Mickeon** [comment]
> I'm not sure my reviews show up for anyone, since I'm not a member and they're not visible when I'm not logged in,

Any user can review, add comments and request changes! It should work completely fine. You may have forgotten to submit your review (there's a big green button on the top right of the files tab), as they are not showing up to anyone.

**Review by ryevdokimov — CHANGES_REQUESTED** [review]

**Comment by ryevdokimov** [comment]
Hah, doy. Thank you.

File: editor/plugins/node_3d_editor_plugin.cpp [hunk 4]
```diff
@@ -8528,7 +8528,8 @@ Node3DEditor::Node3DEditor() {
...
+	settings_vbc->add_margin_child(TTR("Perspective vertical FOV (deg.):"), settings_fov);
```
[the remark this answers is not in the document]
Comment (Mickeon) [reply editor/plugins/node_3d_editor_plugin.cpp:8580]: I agree but for a different reason. I'm more concerned on how much horizontal space the text takes (the other Viewport settings are much shorter in comparison).

**Review by ryevdokimov — APPROVED** [review]
Looks good.

**Review by Mickeon — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**